### PR TITLE
Init project setup (vite with React v19 + Typescript + SCSS)

### PR DESCRIPTION
**Pull request by apyd**
This PR contains init project structure with React v19 with SCSS styles structure and icons needed for project development

It also contains Button component (init version to use and develop based on needs but shows how to use SCSS variables/mixins within components and where to put UI components so smallest bits in project structure)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# kanban-board
\ No newline at end of file
+Kanban board app.
+
+Tech stack:
+- Frontend: Vite, React, Typescript, SCSS, React Query
```
Comment (fukpyd) [inline README.md:4]: please, add more information about local project setup

Comment (fukpyd) [inline README.md:4]: Please, add more information how to run project locally

File: src/components/ui/Button/Button.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@use '../../../scss/variables' as c;
```
Comment (fukpyd) [inline src/components/ui/Button/Button.scss]: Please use alias instead of relative path

File: src/main.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import App from './App.tsx'
+import './scss/global.scss'
```
Comment (fukpyd) [inline src/main.tsx]: please, use alias instead of relative path

File: src/scss/global.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  input,
+  button,
+  textarea,
+  select {
+    font: inherit;
```
Comment (fukpyd) [inline src/scss/global.scss]: please, check out font property: maybe there should be "font-family"?

File: src/scss/variables/_colors.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+// grey tones
+$c-grey-100: #efeffa;
+$c-grey-200: #f4f7fd;
+$c-grey-300: #828fa3;
+$c-grey-400: #2b2c37;
+$c-grey-500: #20212c;
+$c-grey-600: #000112;
+
+// light
```
Comment (fukpyd) [inline src/scss/variables/_colors.scss]: This is already in theme.scss. Please, remove it

**Review by fukpyd — CHANGES_REQUESTED** [review]

File: src/scss/global.scss (hunk 4 above)
Comment (apyd) [reply src/scss/global.scss]: this is correct, font is a shorthand property for setting 'font-style', 'font-variant', 'font-weight', 'font-size', 'line-height', and 'font-family'

File: src/scss/variables/_colors.scss (hunk 5 above)
Comment (apyd) [reply src/scss/variables/_colors.scss]: done

File: src/main.tsx (hunk 3 above)
Comment (apyd) [reply src/main.tsx]: done

File: README.md (hunk 1 above)
Comment (apyd) [reply README.md:4]: done, will add more at later point as this project will grow

File: src/components/ui/Button/Button.scss (hunk 2 above)
Comment (apyd) [reply src/components/ui/Button/Button.scss]: I realized I should use here var(--c-<color-name>) as it is a variable available from :root (html) so it's no longer relevant

**Review by fukpyd — APPROVED** [review]

**Pull request merged**